src/Microsoft.Azure.Functions.Extensions.Mcp/McpToolExtensions.cs

```diff
@@ -74,4 +80,18 @@ internal static JsonElement GetPropertiesInputSchema(this IMcpTool tool)
         using var doc = JsonDocument.Parse(ms);
         return doc.RootElement.Clone();
     }
+
+    private static void WriteEnumValuesIfPresent(Utf8JsonWriter writer, IEnumerable<string>? enumValues)
+    {
+        if (enumValues is not null && enumValues.Any())
```

Comment:
nit: reverse and return early
```suggestion
        if (enumValues is null || !enumValues.Any())
```